modules/ROOT/pages/setting-up-your-dev-environment.adoc

```diff
@@ -73,9 +73,6 @@ image::installnewsoftware.png[InstallNewSoftware]
 +
```

Comment:
I know all you did was remove the note, so this comment is out of scope, but line 74 is really more than one step.

Reply:
Cleaned up, thanks for spotting that!